src/soundio/soundmanager.cpp

```diff
@@ -562,11 +562,11 @@ SoundDeviceError SoundManager::setConfig(const SoundManagerConfig& config) {
     m_pConfig->set(ConfigKey("[Soundcard]","Samplerate"),
                    ConfigValue(static_cast<int>(m_config.getSampleRate())));
 
-    err = setupDevices();
-    if (err == SOUNDDEVICE_ERROR_OK) {
+    status = setupDevices();
```

Comment:
`status` can just be defined here

Reply:
Oh sure, I could do that, though I'm touching these files in #4862 anyway, thus I'd rather fix it there and keep this as pure renaming PR.